profiles/lib/utils.lua

```diff
@@ -0,0 +1,43 @@
+-- Profile functions to implement common algorithms of data processing
+--
+-- You can run a selection you find useful in your profile,
+-- or do you own processing if/when required.
+
+Utils = {}
+
+-- split string 'a; b; c' to table with values ['a', 'b', 'c']
+-- so it use just one separator ';'
+function Utils.string_list_tokens(str)
+  result = {}
+  local idx = 0
+  for s in str.gmatch(str, "([^;]*)") do
+    if s ~= nil and s ~= '' then
+      idx = idx + 1
+      table.insert(result, idx, (s:gsub("^%s*(.-)%s*$", "%1")))
+    end
+  end
+
+  return result
+end
+
+-- same as Utils.StringListTokens, but with many possible separators:
+-- ',' | ';' | ' '| '(' | ')'
+function Utils.tokenize_common(str)
+  result = {}
+  local idx = 0
+  for s in str.gmatch(str, "%S+") do
```

Comment:
If I’m not mistaken, this code splits a string by whitespace; it doesn’t seem to split by other separators like `;` or `(`. Note that an item in `ref` may contain parentheses, for example [`ref=A1(M)` on a way](http://www.openstreetmap.org/way/280766464) or [on a relation](http://www.openstreetmap.org/relation/103623).

Reply:
Yes, you are correct. This is a first version, need to figure out all cases and change this match string